### DISCUSSION TITLE
Libro.fm

### DISCUSSION THREAD

**Discussion by sgspecker** [openaudible/openaudible#1231] [Ideas]
Along with Audible, I have started buying audiobooks from [Libro.fm](https://libro.fm/). Since they provide DRM-free books in zipped MP3 and M4B formats, it would be great if I could easily add books from my Libro library, too.

Thanks Much,
Scott

**Comment by openaudible** [comment]
We like libro.fm! For now, just drag and drop your downloaded M4B files into OpenAudible or use the "Import Files" from the File menu.
